core/cap-0059.md

```diff
@@ -127,7 +127,7 @@ This CAP is aligned with the following Stellar Network Goals:
         }
     ],
     "return": "BytesObject",
-    "docs": "Adds two BLS12-381 G2 points given in bytes format and returns the resulting G2 point in bytes format.",
+    "docs": "Adds two BLS12-381 G2 points given in bytes format and returns the resulting G2 point in bytes format. G2 serialization format: be_encode(X_c1) || be_encode(X_c0) || be_encode(Y_c1) || be_encode(Y_c0), and the most significant three bits of X_c1 are flags i.e. bits(X_c1) = [compression_flag, infinity_flag, sort_flag, bit_380, .. bit_0]",
```

Comment:
How about 'format is' instead of colon? (here and elsewhere in this PR). Also the formula in code format like you have in some above.